awscli/examples/macie2/create-classification-job.rst

```diff
@@ -0,0 +1,52 @@
+**To create a classification job**
+
+The following ``create-classification-job`` example creates a classification job that analyzes objects in the specified S3 bucket for sensitive data. ::
+
+    aws macie2 create-classification-job \
+        --job-type ONE_TIME \
+        --name "ExampleClassificationJob" \
+        --description "Analyze sensitive data in demo bucket" \
+        --s3-job-definition '{
```

Comment:
This parameter value either needs to be collapsed onto the same line, or each line also needs a line continuation mark.